src/dom/dom.js

```diff
@@ -0,0 +1,186 @@
+import {isString, isUndefined, isJQueryObject} from '../utils/types'
```

Comment:
missing `.js` at the end, without my browser cant load it!

Reply:
same on other lines/files e.g. the line here under it

Reply:
Ok, if this can be added to eslint, it would be great